lib/quoters/WebhookQuoter.ts

```diff
@@ -247,16 +265,25 @@ export class WebhookQuoter implements Quoter {
       }
       //if valid quote, log the opposing side as well
       const opposingRequest = request.toOpposingRequest();
-      const opposingResponse = QuoteResponse.fromRFQ({
+      const opposingResponse = await QuoteResponse.fromRFQ({
         request: opposingRequest,
         data: opposite.data,
         type: opposingRequest.type,
         metadata,
       });
+      const validateOpposingPermissionedTokensError = await RFQValidator.validatePermissionedTokens(
+        opposingRequest,
+        opposite.data,
+        opposingRequest.amount,
+        opposingResponse.response.amountOut,
+        provider,
+        this.log
+      );
```

Comment:
What is the desired behavior for the opposing request, since it's not technically the real quote? If the validation is done on the amount of the trade this might cause the opposing quote to fail validation when the real one is ok?

Reply:
That's a good question. It technically could be the case that the opposing is not permissioned. Given the current state of permissioned tokens, I think it'd make sense to skip sending the opposing quote altogether.

Reply:
Going to keep the request there to not make it too complicated. We just wont perform the permission check on the opposing order.